localization/en-us.lua

```diff
@@ -184,6 +184,8 @@ return {
 			b_lobby_options = "LOBBY OPTIONS",
 			b_copy_clipboard = "Copy to clipboard",
 			b_view_code = "VIEW CODE",
+			b_copy_code = "COPY CODE",
+			b_clipboard = "TO CLIPBOARD",
```

Comment:
```suggestion
			b_copy_code_clipboard = { "COPY CODE", "TO CLIPBOARD" },
```

These really shouldn't be two different entries, they should be just like `b_wait_for_host_start` and the like.